helm-charts/verticadb-operator/values.yaml

```
@@ -600,6 +600,15 @@ alloy:
   # Define the number of replicas for the deployment.
   replicaCount: 1
   
+  # Use self-created RBAC resources for Alloy deployment
+  rbac:
+    create: false
+
+  # Use self-created service account for Alloy deployment
+  serviceAccount:
+    create: false
+    name: vdb-op-alloy
```

```suggestion
    name: alloy-vertica-sa
```
"vdb-op" is not always the release name